doc/classes/Resource.xml

```diff
@@ -55,23 +55,11 @@
 				Returns the [RID] of this resource (or an empty RID). Many resources (such as [Texture2D], [Mesh], and so on) are high-level abstractions of resources stored in a specialized server ([DisplayServer], [RenderingServer], etc.), so this function will return the original [RID].
 			</description>
 		</method>
-		<method name="setup_local_to_scene">
+		<method name="setup_local_to_scene" is_deprecated="true">
 			<return type="void" />
 			<description>
 				Emits the [signal setup_local_to_scene_requested] signal. If [member resource_local_to_scene] is set to [code]true[/code], this method is called from [method PackedScene.instantiate] by the newly duplicated resource within the scene instance.
```

Comment:
The description should start with this paragraph: `[i]Deprecated.[/i] <Add here explanation for why / what to use instead.>`